game/audio.cc

```diff
@@ -291,24 +293,24 @@ struct Synth {
 	Synth(Notes const& notes, unsigned int sr) : m_notes(notes), srate(sr) {}
 	void operator()(float* begin, float* end, double position) {
 		static double phase = 0.0;
-		for (float *i = begin; i < end; ++i) *i *= 0.3; // Decrease music volume
-
-		std::vector<float> mixbuf(end - begin);
+		for (float *i = begin; i < end; ++i) *i *= 0.3f; // Decrease music volume
+		std::int64_t size = end - begin;
+		std::vector<float> mixbuf(static_cast<size_t>(size));
```

Comment:
I guess the same kind of check is necessary in order to prevent negative values to be cast in size_t